src/main/java/com/azure/cosmos/examples/analyticalcontainercrud/sync/AnalyticalContainerCRUDQuickstart.java

```diff
@@ -0,0 +1,184 @@
+
+        //  Create container if not exists
+        CosmosContainerProperties containerProperties =
+                new CosmosContainerProperties(containerName, "/lastName");
+
+        // Set analytical store properties
+        containerProperties.setAnalyticalStoreTimeToLiveInSeconds(-1);
+
+        //  Create container with 200 RU/s
+        container = database.createContainerIfNotExists(containerProperties, 200).getContainer();
+
+        logger.info("Done.");
+    }
+
+    // Update container throughput
+    private void updateContainerThroughput() throws Exception {
+        logger.info("Update throughput for container " + containerName + ".");
+
+        // Specify new throughput value
+        container.replaceProvisionedThroughput(400);
+
+        logger.info("Done.");
+    }
+
+    // Container read
+    private void readContainerById() throws Exception {
+        logger.info("Read container " + containerName + " by ID.");
+
+        //  Read container by ID
+        container = database.getContainer(containerName);
+
+        logger.info("Done.");
+    }
+
+    // Container read all
+    private void readAllContainers() throws Exception {
+        logger.info("Read all containers in database " + databaseName + ".");
+
+        //  Read all containers in the account
+        CosmosPagedIterable<CosmosContainerProperties> containers = database.readAllContainers(new FeedOptions());
+
+        // Print
+        String msg="Listing containers in database:\n";
+        for(CosmosContainerProperties containerProps : containers) {
+            msg += String.format("-Container ID: %s\n",containerProps.getId());
+        }
+        logger.info(msg + "\n");
+
+        logger.info("Done.");
+    }
+
+    // Container delete
+    private void deleteAContainer() throws Exception {
+        logger.info("Delete container " + containerName + " by ID.");
+
+        // Delete container
+        CosmosContainerResponse containerResp = database.getContainer(containerName).delete(new CosmosContainerRequestOptions());
+        logger.info("Status code for container delete: {}",containerResp.getStatusCode());
+
+        logger.info("Done.");
+    }
+
+    // Database delete
+    private void deleteADatabase() throws Exception {
+        logger.info("Last step: delete database " + databaseName + " by ID.");
+
+        // Delete database
+        CosmosDatabaseResponse dbResp = client.getDatabase(databaseName).delete(new CosmosDatabaseRequestOptions());
+        logger.info("Status code for database delete: {}",dbResp.getStatusCode());
+
+        logger.info("Done.");
+    }
+
+    // Cleanup before close
+    private void shutdown() {
+        try {
+            //Clean shutdown
+            deleteAContainer();
+            deleteADatabase();
+        } catch (Exception err) {
+            logger.error("Deleting Cosmos DB resources failed, will still attempt to close the client. See stack trace below.");
+            err.printStackTrace();
```

Comment:
error needs to be passed to slf4j logger rather than directing to stderr'
```suggestion
logger.error("Deleting Cosmos DB resources failed, will still attempt to close the client. See stack trace below., e);
```

see this
http://www.slf4j.org/faq.html#paramException